src/components/CommentsList.tsx

```diff
@@ -0,0 +1,21 @@
+import Comment from './Comment';
+import { classes } from '../style/comments.st.css';
+interface IPostOwner {
+    name: string;
+    profilePicture: string;
+    description: string;
+}
+
+const CommentsList = ({ name, description, profilePicture }: IPostOwner) => (
+    <div className={classes.commentsBlock}>
+        <ul className={classes.commentsList}>
+            <Comment
+                isOwnerComment={true}
+                profilePicture={profilePicture}
+                userName={name}
+                content={description}
```

Comment:
is it content or description?

content sounds better imo

Reply:
write another component for the description